client/my-sites/activity/activity-log-item/style.scss

```diff
@@ -234,6 +235,21 @@
 		}
 	}
 
+	.server-actor-icon {
+		/* stylelint-disable-next-line declaration-property-unit-allowed-list */
+		border-radius: 50%;
+		background-color: #23292e;
+		color: #fff;
```

Comment:
Would be better use CSS variables so that Calypso stays themeable and easier to maintain.

Reply:
6d180f97b4be7ca7cea439b762487b69f218241d